packages/frontend/src/hooks/useAuctionState.ts

```diff
@@ -15,23 +13,20 @@ export function useAuctionState(): AuctionState {
   const { account, chainId } = useEthers()
   const { networks } = useConfig()
   const { state } = useContractState()
-  const [contractState, setContractState] = useState(state)
-
-  useEffect(() => setContractState(state), [state])
 
-  if (contractState === ContractState.AWAITING_BIDDING) {
-    return 'AwaitingBidding'
+  if (state === ContractState.AWAITING_BIDDING) {
+    return getStateUsingWallet(account, chainId, networks, 'AwaitingBidding')
```

Comment:
Why do we want to hide this message when wallet is not connected?

Reply:
Yes, we shouldn't. Reverted changes for `AwaitingBidding` and `AwaitingResults` states.